src/main/java/project/demo/product/product/entity/SaleProductInfo.java

```diff
@@ -41,4 +42,32 @@ public SaleProductInfo(ProductRegistrationRequest productRegistrationRequest, Pr
         this.productInfo = productInfo;
         productInfo.addSalesProduct(this);
     }
+
+    /**
+     * 재고 변경
+     * @param quantity
+     */
+    public void changeStock(int quantity) {
+        if(quantity < 0){
+            throw new ValidationException("재고 수량은 0보다 작을 수 없습니다.");
+        }
+        this.stock = quantity;
+    }
+
+    /**
+     * 매진 여부
+     * @return
+     */
+    public boolean isSoldOut(){
+        return this.stock == 0;
```

Comment:
0 이하인 경우 매진이라고 판단해야 할 것 같음.

```suggestion
        return this.stock <= 0;
```